lib/nerves_init_gadget/network_manager.ex

```diff
@@ -31,6 +31,8 @@ defmodule Nerves.InitGadget.NetworkManager do
     handle_ip_update(state, new_ip)
   end
 
+  defp handle_ip_update(state, nil), do: {:noreply, state}
```

Comment:
Note to self: I don't think this is actually the right way to handle it, but I stuck this in here when I was seeing crashes related to using the `:static` address method but the interface wasn't up yet, or something like that.
Maybe just remove it from this PR and revisit separately in the future if it comes up again.